test/api/peer.transactions.collision.js

```diff
@@ -42,7 +42,7 @@ describe('POST /peer/transactions', function () {
 
 				postTransaction(transaction, function (err, res) {
 					node.expect(res.body).to.have.property('success').to.be.not.ok;
-					node.expect(res.body).to.have.property('message').to.equal('Invalid sender public key: b26dd40ba33e4785e49ddc4f106c0493ed00695817235c778f487aea5866400a expected: ce33db918b059a6e99c402963b42cf51c695068007ef01d8c383bb8a41270263');
+					node.expect(res.body).to.have.property('message').to.equal('Failed to verify signature');
```

Comment:
After the talk with Oliver we came to the conclusion, that the result of that transaction insertion is changed because the tests which are not fired now were changing the state of DB table. This is probably true- the commit (https://github.com/liskHQ/lisk/commit/1fc0f03b1538b7f3fc1f810483428e9869561d43) that changed this expectation was probably added after introducing new test in a different test suite, that mutated db state before.